src/gengowatcher/browser_worker/client.py

```
@@ -0,0 +1,99 @@
+        except RuntimeError:
+            return asyncio.run(self.send_command(payload))
+        return self._submit_from_running_loop(payload)
+
+    def _submit_from_running_loop(self, payload: dict[str, Any]) -> dict[str, Any]:
+        result: dict[str, Any] = {}
+        error: list[BaseException] = []
+
+        def runner() -> None:
+            try:
+                result["value"] = asyncio.run(self.send_command(payload))
+            except BaseException as exc:  # pragma: no cover - defensive propagation
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `BaseException`

(BLE001)